components/home/Newsletter.vue

```diff
@@ -18,7 +18,7 @@
               size="6.4rem"
               gradient="linear-gradient(121.85deg, #C4C4C4 29.11%, #FFFFFF 69.16%)"
             >
-              1000’s
+              1000's
```

Comment:
Why was this changed (single curly quote to single straight quote)? Is this intentional or coming from a formatter?

I think we should prefer the curly quote for this usage. See this [guide](https://practicaltypography.com/straight-and-curly-quotes.html) for details.

Reply:
Replaced it to be consistent with content on other places, In all our microsites and websites we are using single straight quote